dev_guide/secrets.adoc

```diff
@@ -181,7 +181,7 @@ but create new ones with distinct names.
 [[secrets-in-volumes-and-env-vars]]
 == Secrets in Volumes and Environment Variables
 
-See xref:secrets-examples[examples] of YAML files with secret data.
+If your container uses a `secret` in an environment container, you must restart the container to see the updated `secret`. See xref:secrets-examples[examples] of YAML files with secret data.
```

Comment:
Not sure what does `in an environment container` mean, seems not good wording. Maybe just:
```
If your container uses a `secret` as environment variable
```

Reply:
Done